src/styles/index.scss

```diff
@@ -1,3 +1,87 @@
-body {
+@import './variables';
+
+.page {
   margin: 0;
+  box-sizing: border-box;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-flow: wrap;
+  flex-direction: row;
+
+  width: 800px;
+  min-width: 50vmin;
+  max-width: 100vmin;
+
+  padding: 10px;
+  gap: 1px;
+
+  &__day {
+    width: $box-size;
+    height: $box-size;
+    background-color: #eee;
+    border: 1px solid black;
+
+    font-size: 30px;
+    font-family: Arial, sans-serif;
+
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    position: relative;
+    cursor: pointer;
+
+    transition: 0.5s background-color;
+
+    &__day {
```

Comment:
It seems there is a mistake in the BEM naming convention. The element `calendar__day` is nested within itself which is incorrect. You should remove the nested `.calendar__day` and apply styles directly to `.calendar__day`.